json-c.sh

```diff
@@ -1,33 +1,19 @@
 package: json-c
-version: "v0.13.1"
-tag: "json-c-0.13.1-20180305"
+version: "v0.17.0"
+tag: "json-c-0.17-20230812"
 source: https://github.com/json-c/json-c
 build_requires:
-  - "autotools:(slc6|slc7)"
+  - CMake
   - "GCC-Toolchain:(?!osx)"
+  - alibuild-recipe-tools
 ---
 #!/bin/bash -e
 rsync -a --delete --exclude '**/.git' --delete-excluded $SOURCEDIR/ ./
-autoreconf -ivf
-./configure --disable-shared --enable-static --prefix="$INSTALLROOT"
+
+mkdir build
+cd build
+../cmake-configure --disable-shared --enable-static --prefix="$INSTALLROOT"
```

Comment:
Shouldn't we simply use cmake? It would also allow us to get rid of the rsync above.